spec/.babelrc

```diff
@@ -3,12 +3,12 @@
         "@babel/plugin-proposal-object-rest-spread"
     ],
     "presets": [
+        "@babel/preset-typescript",
         ["@babel/preset-env", {
             "targets": {
-                "node": "14"
+                "node": "18"
```

Comment:
Is this a breaking change?

Reply:
No, not as far as I know. We already only support Node 18+